README.md

```diff
@@ -49,8 +49,7 @@ See [Release Notes for R5](Release5.md) for more information.
 ## Roadmap
 
 We have a 6 month release cadence -- R6 will follow in March 2024.
-Until then, we will provide bugfixes and security fixes for R.
+Until then (and a bit beyond), we will provide bugfixes and security fixes for R5.
```

Comment:
I find the "(and a bit beyond)" difficult. Either we specify it or we leave it out.

```suggestion
Until then, we will provide bugfixes and security fixes for R5.
```

Reply:
Until then (and 6 weeks beyond), we will provide ... ?
This is what we currently commit to.